packages/manager/modules/product-offers/src/workflows/product-offers-order-workflow.class.js

```diff
@@ -68,16 +69,8 @@ export default class OrderWorkflow extends Workflow {
    * @param {Object} WucOrderCartService Service to handle order cart
    */
   /* @ngInject */
-  constructor(
-    locale,
-    $q,
-    $timeout,
-    $translate,
-    workflowOptions,
-    WucOrderCartService,
-  ) {
-    super(locale, $q, $translate, workflowOptions);
-    this.$timeout = $timeout;
+  constructor($q, $timeout, $translate, workflowOptions, WucOrderCartService) {
```

Comment:
As `locale` was the first parameter and even if uses were updates, this looks like a `BREAKING CHANGE`

Reply:
IMO, it would be breaking if the updates were not embedded in the same commit.
If you cherry-pick the related commit, the app is no more functioning, so it isn't breaking then, what do you think ?

Reply:
@JeremyDec Imo we should have two commits : one with the breaking change and the other one fixing usage (even though package is private, if it was published it would be best for other users to know the new version is breaking)

Reply:
@marie-j this is where I disagree. I don't think it is a good practice to force a breaking change here. 
The changes and its fixes, are all located in the module, and won't change the usage.

Reply:
Could the class used here be used outside of the module ? If the answer is yes then it's breaking, if not then it's fine

Reply:
Yes the workflow class are used only inside the module